core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java

```diff
@@ -938,7 +938,12 @@ public void close() {
     public static final String CAPACITY_FREE = "CapacityFree";
     public static final String WORKERS = "Workers";
 
-    private static void registerGauges(final BlockMaster master) {
+    /**
+     * Registers metric gauges.
+     *
+     * @param master the block master handle
+     */
+    public static void registerGauges(final BlockMaster master) {
```

Comment:
Visible for testing?

Reply:
this is not only visible for testing, as it is also called upon registration.

Reply:
Visible for testing can mean the visibility (ie. `public`) is unnecessarily permissive in order to make testing easier. Isn't this the case?